### ISSUE TITLE
UpdateContentBuilder does not build script

### ISSUE THREAD

**Issue by ngocduy-pham** [sksamuel/elastic4s#930]
Using elastic4s http, here's the log:

    [info] 2017-05-28 07:24:33,703 DEBUG com.sksamuel.elastic4s.http.bulk.BulkImplicits$BulkExecutable$ - Executing elastic request POST:/_bulk?
    [info] 2017-05-28 07:24:33,703 DEBUG com.sksamuel.elastic4s.http.bulk.BulkImplicits$BulkExecutable$ - {"update":{"_index":"static","_type":"platform_role","_id":"AAAAAAAAwAIAEAAAAAAAAA"}}
    [info] {}
    [info] {"update":{"_index":"static","_type":"platform_role","_id":"AAAAAAAAwAMAEAAAAAAAAA"}}
    [info] {}
    [info] 2017-05-28 07:24:33,705 DEBUG org.elasticsearch.client.RestClient - request [POST http://localhost:9200/_bulk] returned [HTTP/1.1 400 Bad Request]

In the body of `com.sksamuel.elastic4s.http.update.UpdateContentBuilder.apply`, no access to `request.script`.

**Comment by ibaisi** [comment]
I'm having the same issue. When trying to write an update such as:

`val qry = update(groupId).in(AwkProfileConfig.ESConfig.index / AwkProfileConfig.ESConfig.groupType) script "ctx._source.userCount += 1"
      println("UPDATE QUERY: " + client.show(qry))`

I get back:

`UPDATE QUERY: {}`

**Comment by vfilatov-naumen** [comment]
As temporary solution one can implement their own HttpExecutable class:

```scala
    case class RawUpdateHttpExecutable(script: String) extends HttpExecutable[UpdateDefinition, UpdateResponse] with Logging {

        override def execute(client: RestClient,
                             request: UpdateDefinition): Future[UpdateResponse] = {

            val method = "POST"
            val endpoint = s"/${request.indexAndTypes.index}/${request.indexAndTypes.types.mkString(",")}/${request.id}/_update"

            val params = scala.collection.mutable.Map.empty[String, Any]
            request.fetchSource.foreach { context =>
                if (!context.fetchSource) params.put("_source", "false")
            }
            request.retryOnConflict.foreach(params.put("retry_on_conflict", _))
            request.parent.foreach(params.put("parent", _))
            request.routing.foreach(params.put("routing", _))
            request.refresh.map(RefreshPolicyHttpValue.apply).foreach(params.put("refresh", _))
            request.version.map(_.toString).foreach(params.put("version", _))
            request.versionType.foreach(params.put("version_type", _))
            request.waitForActiveShards.foreach(params.put("wait_for_active_shards", _))

            val entity = new StringEntity(script, ContentType.APPLICATION_JSON)
            logger.debug(s"Update Entity: $script")

            client.async(method, endpoint, params.toMap, entity, ResponseHandler.default)
        }
    }
```
And then use it like:
```scala
    client.execute {
        update(id).in(index)
    }(RawUpdateHttpExecutable(myUpdateScript))
```

**Comment by sksamuel** [comment]
Fixed in 5.4.4
